File: tests/conformance/fixtures/go/14-gate-file.go
package main

import sykli "github.com/false-systems/sykli-elixir/sdk/go"

func main() {
	p := sykli.New()

	p.Gate("wait-approval").
		GateStrategy("file").
		GateTimeout(1800).
		GateFilePath("/tmp/approved")

	p.Emit()
}

File: tests/conformance/fixtures/elixir/14-gate-file.exs
use Sykli

pipeline do
  gate "wait-approval" do
    gate_strategy "file"
    gate_timeout 1800
    gate_file_path "/tmp/approved"
  end
end
|> Sykli.Emitter.validate!()
|> Sykli.Emitter.to_json()
|> IO.puts()
